Formula/cpanminus.rb

```diff
@@ -1,15 +1,13 @@
 class Cpanminus < Formula
   desc "Get, unpack, build, and install modules from CPAN"
   homepage "https://github.com/miyagawa/cpanminus"
-  url "https://github.com/miyagawa/cpanminus/archive/1.9019.tar.gz"
-  sha256 "d0a37547a3c4b6dbd3806e194cd6cf4632158ebed44d740ac023e0739538fb46"
+  # Don't use git tags, their naming is misleading
```

Comment:
any open thread about this git tag discussion?

Reply:
Didn't find anything